storage.bs

```diff
@@ -178,7 +178,18 @@ larger <a>site storage quota</a>. Factors such as navigation frequency, recency
 bookmarking, and <a href="#persistence">permission</a> for {{"persistent-storage"}} can be used as
 indications of "popularity".
 
+The <dfn export>application cache site storage usage</dfn> for an origin is a rough estimate of the amount
+of bytes used in Application Cache in <var>origin</var>'s <a>site storage unit</a>. [[HTML#appcache]]
 
+The <dfn export>caches site storage usage</dfn> for an origin is a rough estimate of the amount
+of bytes used in Cache API in <var>origin</var>'s <a>site storage unit</a>. [[service-workers-1#cache-objects]]
```

Comment:
Reference `[[SERVICE-WORKERS]]`, not `[[SERVICE-WORKERS-1]]`, if possible. (It might not be; there have been weird bugs in the past.)